packages/grafana-ui/src/utils/displayValue.ts

```diff
@@ -67,7 +74,8 @@ export function getDisplayProcessor(options?: DisplayValueOptions): DisplayProce
 
       if (!isNaN(numeric)) {
         if (shouldFormat && !_.isBoolean(value)) {
-          text = formatFunc(numeric, options.decimals, options.scaledDecimals, options.isUtc);
+          const decimalInfo = getDecimalsForValue(value);
```

Comment:
needs a if (!options.decimals) so the auto decimals is only done if user has not specified any decimals. Think a unit test case for this would nice (among the getDecimalProcessor tests)

Reply:
Ooops